### ISSUE TITLE
please support webtoon bestChallenge and challenge

### ISSUE THREAD

**Issue by gfhghfghfh** [Xonshiz/comic-dl#236]
https://comic.naver.com/genre/bestChallenge.nhn
https://comic.naver.com/genre/challenge.nhn

**Comment by Xonshiz** [comment]
Naver is already supported. You should be able to pass the link of the comic to the tool and it'll download all the images in it.

**Comment by gfhghfghfh** [comment]
@Xonshiz

I know https://comic.naver.com/webtoon/weekday is already supported,
but webtoon bestChallenge and challenge is not supported.

Can you click on the url i gave you to see?

https://comic.naver.com/genre/bestChallenge.nhn
https://comic.naver.com/genre/challenge.nhn

ex:
https://comic.naver.com/bestChallenge/list?titleId=791651
https://comic.naver.com/challenge/list?titleId=678448

**Comment by Xonshiz** [comment]
Just look at the code. Well, this should've worked. But, looks like Naver has changed their URL scheme, that's the reason this isn't going through. Either they changed their URL scheme and whole Naver download is broken or they introduced a new scheme as well.

Anyways, thanks for bringing this to notice. I'm re-opening this issue and will push a fix for this by tomorrow probably. Thanks for waiting such a long time and apologies :)

**Comment by gfhghfghfh** [comment]
thanksRegarding the issues mentioned above
"But, looks like Naver has changed their URL scheme"

I list some examples


***not working***
comic_dl.exe -i https://comic.naver.com/bestChallenge/list.nhn?titleId=789786
comic_dl.exe -i https://comic.naver.com/bestChallenge/list?titleId=789786
comic_dl.exe -i https://comic.naver.com/webtoon/list?titleId=670152


***working***
comic_dl.exe -i https://comic.naver.com/webtoon/list.nhn?titleId=670152

**Comment by Xonshiz** [comment]
Looks like new scheme got added as well.
I picked these from `comic.naver.com` Homepage itself:
https://comic.naver.com/webtoon/list?titleId=775318
https://comic.naver.com/webtoon/list?titleId=774044


These are the newly added URL schemes.

Anything with `list.nhn` or `details.nhn` will work.
I'll add support for these new ones too. Then your challenge and bestChallenge ones should work too :)

Cool, I'll try to push the new changes by tomorrow (or in next few hours).
